pages/admin/permit-holders.tsx

```diff
@@ -63,6 +52,9 @@ const COLUMNS = [
     disableSortBy: true,
     width: 140,
     maxWidth: 140,
+    Cell: ({ value }: any) => {
+      return <Text>{new Date(value).toLocaleDateString('en-ZA')}</Text>;
```

Comment:
What is `en-ZA`?

Reply:
`"yyyy/MM/dd"`

https://stackoverflow.com/questions/2388115/get-locale-short-date-format-using-javascript#:~:text=%22en-za%22%3A%20%22yyyy%2Fmm%2Fdd%22%2C